src/lib/containers/GenericCard.tsx

```diff
@@ -153,11 +153,35 @@ export const renderLabel = (args: {
     columnModels,
   })
 
-  if (!labelLink || !str) {
-    // they either don't need a link or the array came back empty
+  if (!str) {
+    // the array came back empty
     return str
   }
 
+  let className = ''
+  const style: React.CSSProperties = {}
+  if (isHeader) {
+    className = 'SRC-lightLink'
+  } else {
+    className = 'SRC-primary-text-color'
+  }
+
+  if (!labelLink) {
+    // if this looks like a Synapse ID, then autolink.
+    if (str.match(SYNAPSE_REGX)) {
+      // its a synId
+      return <a
+          target="_blank"
+          rel="noopener noreferrer"
+          href={`https://www.synapse.org/#!Synapse:${str}`} className={className} style={style}>
```

Comment:
The style object seems to be empty here. Should you delete it?

Reply:
Deleted!